spec/integration/user_filters_spec.rb

```diff
@@ -168,6 +168,18 @@
     expect(pretty).to eq [[ConnectionHelper::DEFAULT_USERNAME, "[Dev] IN ([tomas@gooddata.com])"]]
   end
 
+  it 'works with filters from .get_filters' do
+    filters = GoodData::UserFilterBuilder.get_filters(
+      File.expand_path('../../data/user_filters.csv', __FILE__),
+      :type => :filter,
+      :labels => [{ :label => @label, :column => 'filter' }]
+    )
+    @project.add_data_permissions(filters)
+    permissions = @project.data_permissions
+    pretty = permissions.first.pretty_expression
```

Comment:
does this have to be order dependent? wouldn't it be safer to check if the count of the permissions is 2 and one of them is tomas@gooddata.com ?